db/migrate/20190109112643_add_size_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddSizeToTags < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tags, :size, :integer
```

Comment:
What's the default for this? As in, how do you know that for a given tag we have never computed its size?

Reply:
I thought about leaving null but it can also be 0. To be honest it doesn't matter because we don't care to whatever is there. We just compute it somewhere and overwrite the value whenever we need to. Now, if it's null we display `-` as you can see in the screenshot.